saos-webapp/src/main/webapp/WEB-INF/view/details/judgmentDetailsNavigation.jsp

```diff
@@ -283,7 +283,7 @@
             <div class="legalBases">    
                 <c:forEach items="${judgment.legalBases}" var="legalBase" >
                     <div class="legalBase">
-                        <a href="${contextPath}/search?legalBase=${legalBase}" data-toggle="tooltip" title="${legalBasesLinkTooltip}" >
+                        <a href="${contextPath}/search?legalBase=%22${legalBase}%22" data-toggle="tooltip" title="${legalBasesLinkTooltip}" >
```

Comment:
I think we should encode the url? What if legalBase will contain '?' for example?